aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -119,16 +118,21 @@ macro_rules! unwrap_or_discard {
 }
 
 impl AptosVM {
-    pub fn new(state: &impl StateView) -> Self {
-        Self(AptosVMImpl::new(state))
+    pub fn new(config_storage: &impl ConfigStorage) -> Self {
```

Comment:
What is `ConfigStorage`? How is it different from `StateView`? Why is `ConfigStorage` enough here instead of `StateView`?

Reply:
`ConfigStorage` is an abstraction which allows you to get resources which are configs, e.g. gas schedule, etc. In case of VM, we don't care about state view when we initialise, all we care is the configs.